app/redhat_insights/default/data/ui/views/main_dashboard.xml

```diff
@@ -869,6 +869,71 @@
 					}
 				}
 			]
+		},
+		"viz_8eAN6hh5": {
+			"type": "splunk.pie",
```

Comment:
This type does not work on my side. Shouldn't it be `"type": "viz.pie"`?

![Screenshot 2022-09-14 at 17-05-42 Dashboard](https://user-images.githubusercontent.com/7695766/190192638-fba4c641-53cb-4a0e-9b04-9396fa0bf03c.png)

Reply:
weird... I just set the Pie chart on object and splunk set as `splunk.pie`.. tested here changing to `viz.pie` and works for me too. 

![image](https://user-images.githubusercontent.com/2904206/190210400-434af878-3543-4e51-abc4-612162464166.png)

I changed the code.